packages/yoga/src/Checkbox/web/Checkbox.jsx

```diff
@@ -250,6 +250,8 @@ const Checkbox = ({
 }) => {
   const inputRef = useRef(null);
 
+  const { onChange, ...restWithoutOnChange } = rest;
```

Comment:
Hello, with this change the "Checkbox" called now can't perform a custom action when the checkbox changes... Right? I don't see `onChange` being passed anywhere...

Ex: "I want to show/hide another field when I check the box"